JenkinsFile

```diff
@@ -158,7 +158,11 @@ node {
 					  cf set-env ${appName} DOMAIN ${env.PHASE_ONE_PCF_DOMAIN}
 					  cf set-env ${appName} OSM_BASE_URL ${env.OSM_BASE_URL}
 					  cf set-env ${appName} PLANET_BASE_URL ${env.PLANET_BASE_URL}
+					  cf set-env ${appName} USER_GUIDE_URL "${env.USER_GUIDE_URL}"
```

Comment:
I'm guessing that it is required, just the same as the other environment variables listed there.  The only difference is that this one may not be set, which is why I have the "s around the value; that way if it isn't set then it won't break the syntax of the `cf set-env` command (I'm just guessing here since I couldn't test it).  Instead of breaking the syntax, the environment variable will be set to a zero-length string, which the application will treat the same way as a `null` or `undefined` value.

Reply:
I guess i mean wont it be set twice then.  Once in line 161 and again on line 164 (if USER_GUIDE_URL is populated)